app/src/main/kotlin/com/simplemobiletools/smsmessenger/extensions/Math.kt

```diff
@@ -0,0 +1,8 @@
+package com.simplemobiletools.smsmessenger.extensions
+
+import kotlin.math.roundToInt
+
+/**
+ * Returns the closest next number divisible by [multipleOf].
+ */
+fun Int.round(multipleOf: Int = 1) = (toDouble() / multipleOf).roundToInt() * multipleOf
```

Comment:
lets use another name for it, Round it something else

Reply:
done